src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const ONE_DAY_COST = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const DEFAULT_PRICE = ONE_DAY_COST * days;
```

Comment:
```suggestion
  const currentPrice = ONE_DAY_COST * days;
```
Variable names should be in camelCase